tutorials/MixedHMC.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>Feel free to subordinate this to a later PR: At some point, it would be great if we can add documentation explaining what Mixed HMC is and how to use it to our documentation site. It might also be useful to add some remarks to this tutorial, helping to explain to the user what MixedHMC is, when/why they might want to use it, and a bit about how/why you decided to use the dataset that you have used in this tutorial. Of course, it might be easier to write all of this at the same time, so happy for that to happen in a later PR if you think that's best!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/facebookresearch/beanmachine/pull/1491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I think some basic documentation should be part of this PR. Not necessarily a full tutorial, but at least enough to get someone started.